apps/wallet/src/ui/app/hooks/deepbook/useGetEstimate.ts

```diff
@@ -156,12 +149,13 @@ async function getPlaceMarketOrderTxn({
 			);
 		}
 
-		const walletFee = new BigNumber(quoteBalance)
-			.times(WALLET_FEES_PERCENTAGE / 100)
-			.integerValue(BigNumber.ROUND_DOWN)
-			.toString();
+		const actualBalance = isPayAll
+			? new BigNumber(quoteBalance).minus(getWalletFees(quoteBalance))
```

Comment:
similar here - also it seems the code is almost the same, we could probably move it out of the ifs right?

Reply:
refer to above